cypress/e2e/specs/check_cart_price.cy.ts

```diff
@@ -0,0 +1,15 @@
+import CartPage from "cypress/support/pages/cart_page";
+
+describe("Cart Price Check Test Suite", () => {
+  it("user adds first product to cart and checks prices and totals", () => {
```

Comment:
Please rebase and move to https://github.com/abdullah-aladham/automationStore-CypressTS/blob/master/cypress/e2e/specs/add-product-to-cart-spec.cy.ts